Electrometer/v4/summit_comcam.yaml

```diff
@@ -0,0 +1,13 @@
+# Configuration for photodiode electrometer attached to comcam cart.
```

Comment:
So the issue here is that there is a mapping between CSC index and the config file. I'm hoping @wvreeven knows what I'm talking about... otherwise I can chase it down.

Reply:
Yes we had a similar challenge with the ESS CSC. Supporting this in the CSC requires quite a change in the python code though. Maybe something for a separate ticket?

Reply:
I also did this with the PMD. I think that mapping will require a separate ticket otherwise it will make this ticket rather large.

Reply:
I'm ok to do this under a separate ticket, but we should do it in short order as these devices will hit the summit next week.